alkira/resource_alkira_connector_azure_vnet.go

```diff
@@ -403,12 +406,10 @@ func resourceConnectorAzureVnetDelete(ctx context.Context, d *schema.ResourceDat
 
 // generateConnectorAzureVnetRequest generate request for connector-azure-vnet
 func generateConnectorAzureVnetRequest(d *schema.ResourceData, m interface{}) (*alkira.ConnectorAzureVnet, error) {
-
 	//
 	// Segment
 	//
 	segmentName, err := getSegmentNameById(d.Get("segment_id").(string), m)
-
```

Comment:
not sure why all those lines change, seems like not related. Please keep the change related to this PR, so it's easier to see what's the real change.